run.go

```diff
@@ -0,0 +1,198 @@
+package pack
+
+import (
+	"context"
+	"crypto/md5"
+	"fmt"
+	"io"
+	"log"
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	"github.com/buildpack/pack/config"
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/go-connections/nat"
+	"github.com/pkg/errors"
+)
+
+type RunFlags struct {
+	AppDir   string
+	Builder  string
+	RunImage string
+	Port     string
+}
+
+type RunConfig struct {
+	Port  string
+	Build Task
+	// All below are from BuildConfig
+	AppDir   string
+	Builder  string
+	RunImage string
+	RepoName string
+	// Above are copied from BuildFlags are set by init
```

Comment:
I find the "Above" comments a little confusing, especially with the "All below" comment at the top.

Reply:
Would you believe that I copied that comment from BuildConfig, which the first comment is trying to describe? Yea, it's confusing, I'll remove the extra comments